pragma/file.go

```diff
@@ -45,7 +73,60 @@ func readLine(reader *bufio.Reader) (string, error) {
 	return lineBuf.String(), nil
 }
 
-func (f *File) extractPragmas() {
+// issueCodeRegex matches issue codes of the format XXX-XX1234.
+// See: https://regex101.com/r/4PIOIi/1
+var issueCodeRegex = regexp.MustCompile(`\w+-\w{0,2}\d{1,4}`)
```

Comment:
This locks the library to an issue code format, don't do that.

Instead:
- For checking if the filename is an issue code, just check if the file name is present in analysis_results file.
- Simply assume the scatr pragmas to be correct, don't validate their issue codes to abide to a format.

This way, other people can make up whatever issue code format they want (like `my-issue-slug` instead of `XY-Z1234`).